test/system/src/test/java/io/pravega/test/system/AbstractFailoverTests.java

```diff
@@ -194,11 +194,6 @@ private void closeWriter(EventStreamWriter<Long> writer) {
                         log.debug("Read timeout");
                     }
                 } catch (Throwable e) {
```

Comment:
During a failover scenario `e` can be an instance of RetriesExhaustedException . (We have observed this during yesterday's system test run.)